ui/src/dashboards/components/DashboardPage.tsx

```diff
@@ -101,10 +110,6 @@ const mstp = (state: AppState) => {
   }
 }
 
-const mdtp = {
-  resetCachedQueryResults: resetCachedQueryResults,
```

Comment:
and this added back

Reply:
Not unless we want to integrate the old system back in before we end up removing it later?